.github/workflows/portal-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Portal CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('context/requirements.txt') }}
+
+    - name: Set up Node 12.16.3
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.3'
+
+    - name: Set up git config to make .dev-start happy
+      run: git config --global submodule.recurse true
```

Comment:
Huh... `submodules: 'recursive'` above isn't sufficient?

Reply:
Isn't it necessary since `dev-start` has the below?

```
[ "`git config --get submodule.recurse`" == 'true' ] \
  || die "To update submodules automatically: git config --global submodule.recurse true";
```

Reply:
Got it. I think you're right: What you're doing is the simplest way to do this.